numba_dppy/tests/passes/__init__.py

```diff
@@ -1,27 +1,21 @@
-#! /usr/bin/env python
-# Copyright 2021 Intel Corporation
+################################################################################
+#                                 Numba-DPPY
+#
+# Copyright 2020-2021 Intel Corporation
 #
 # Licensed under the Apache License, Version 2.0 (the "License");
 # you may not use this file except in compliance with the License.
 # You may obtain a copy of the License at
 #
-#      http://www.apache.org/licenses/LICENSE-2.0
+#    http://www.apache.org/licenses/LICENSE-2.0
 #
 # Unless required by applicable law or agreed to in writing, software
 # distributed under the License is distributed on an "AS IS" BASIS,
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
+################################################################################
 
-import pytest
-
-offload_devices = [
-    "opencl:gpu:0",
-    "level_zero:gpu:0",
-    "opencl:cpu:0",
-]
-
+from numba_dppy.tests.njit_tests.dpnp import *
```

Comment:
If it is for skipping because dpnp is absent, then it is strange. Passes tests are not all depend on dpnp.